parts/linux/cloud-init/artifacts/components.json

```diff
@@ -701,7 +701,7 @@
             "versionsV2": [
               {
                 "renovateTag": "<DO_NOT_UPDATE>",
```

Comment:
we have this set as a static version - can we remove this to have renovate start updating this automatically?

Reply:
also @tyler-lloyd, you'll need to re-open this PR without using a fork